phpBB/styles/prosilver/template/search_results.html

```diff
@@ -115,16 +131,14 @@ <h2 class="searchresults-title"><!-- IF SEARCH_TITLE -->{SEARCH_TITLE}<!-- ELSE
 							<!-- ENDIF -->
 
 							<div class="responsive-hide left-box">
-								<!-- IF searchresults.S_HAS_POLL --><i class="icon fa-bar-chart fa-fw" aria-hidden="true"></i><!-- ENDIF -->
-								<!-- IF searchresults.ATTACH_ICON_IMG --><i class="icon fa-paperclip fa-fw" aria-hidden="true"></i><!-- ENDIF -->
-								{% EVENT topiclist_row_topic_by_author_before %}
```

Comment:
This is the reason tests are failing, you've removed this `_before` and `_after` event.

Reply:
lets see